source/Module/Common/Functions.XccdfXml.ps1

```diff
@@ -528,6 +527,11 @@ function Get-SqlTechnologyRole
     $split = $Path -split '_'
     $stigIndex = $split.IndexOf('STIG')
     $sqlRole = $split[$stigIndex -1]
+    if ($sqlRole -match '\w\d{1,}\w\d{1,}')
+    {
+        $null = $Id -match "(?<Type>Database|Instance)"
+        $sqlRole = $Matches['Type']
+    }
```

Comment:
Should there be the extra spaces in the get-stigversionNumber function below (line 562)

Reply:
extra spaces and line returns removed, good catch